src/Symfony/Bundle/FrameworkBundle/composer.json

```
@@ -48,6 +48,7 @@
         "symfony/messenger": "^4.4|^5.0",
         "symfony/mime": "^4.4|^5.0",
         "symfony/process": "^4.4|^5.0",
+        "symfony/scheduler": "^4.4|^5.0",
```

shouldn't be 5.1 ?

---

```suggestion
        "symfony/scheduler": "^5.2",
```

There is no version 4.4 of that package.